apps/sharedtest/test/unit/dialer/font_size_manager_test.js

```diff
@@ -223,6 +223,23 @@ suite('font size manager', function() {
         });
       });
 
+      suite('on <div> with bdi element', function() {
```

Comment:
These tests are insufficient. We should be repeating several of the key tests above, but on an element with a ```<bdi>``` tag.